engine/src/test/java/com/redhat/cloud/notifications/db/repositories/DrawerNotificationRepositoryTest.java

```diff
@@ -63,32 +70,67 @@ void removeFixtures() {
         entityManager.createQuery("DELETE FROM DrawerNotification").executeUpdate();
     }
 
+    private void createDrawerNotification(DrawerNotification drawerNotification) {
+        entityManager.persist(drawerNotification);
+    }
+
     @Test
     @Transactional
     void testSimpleCreateGetCascadeDelete() {
```

Comment:
Now that the old `DrawerNotificationRepository#create` method has been removed, this test only tests test code so we don't really need it anymore. That's a lot of "test" in the same sentence 😅.

Reply:
Okay let's keep it for now.